config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
 end
```

Comment:
- [ ] Great job so far, but we have some minor issues to fix before you can move forward. You have defined separate routes for users and posts, without any nesting. It creates routes for users and posts as individual resources. The routes generated would be something like /users and /users/:id for users, and /posts and /posts/:id for posts.
As per requirements, we are to have nested routes, where a can have user has many posts. This will create routes for users and their associated posts. The routes generated would be something like /users/:user_id/posts and /users/:user_id/posts/:id, allowing you to access posts belonging to a specific user. See below:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```